Source/monster.cpp

```diff
@@ -479,6 +475,8 @@ void PlaceUniqueMonst(int uniqindex, int miniontype, int bosspacksize)
 		UberDiabloMonsterIndex = ActiveMonsterCount;
 	}
 	PlaceMonster(ActiveMonsterCount, uniqtype, position);
+	ActiveMonsterCount++;

```

Comment:
Was this some sort of bug?

Reply:
PrepareUniqueMonster was incrementing ActiveMonsterCount in response to the allocation in this function. This meant that the use in debug.cpp needed to compensate for it when spawning uniques as part of the debug command. PrepareUniqueMonster shouldn't modify ActiveMonsterCount at all so I moved the increment here.